
# LETTERS
> 3′ UTR shortening represses tumor-suppressor genes in trans by disrupting ceRNA crosstalk
> 翻译: 3’UTR缩短，通过干扰ceRNA交谈，反式抑制抑癌基因。
> Letter | Published: 21 May 2018
> Nature Genetics volume 50, pages783–789 (2018)
> Hyun Jung Park, Ping Ji, Soyeon Kim, Zheng Xia, Benjamin Rodriguez, Lei Li, Jianzhong Su, Kaifu Chen, Chioniso P. Masamha, David Baillat, Camila R. Fontes-Garfias, Ann-Bin Shyu, Joel R. Neilson, Eric J. Wagner & Wei Li 
> https://www.nature.com/articles/s41588-018-0118-8
> https://pubmed.ncbi.nlm.nih.gov/29785014/
> 新闻稿: https://www.sohu.com/a/253459835_769248


# Abstract

![0521-NG-3UTR-ceRNA-fig3c.png](/data/2021/images/05/0521-NG-3UTR-ceRNA-fig3c.png)
Fig.3c, 3′US genes (left) might repress their ceRNA partner PTEN in trans through miRNAs (middle) commonly released through 3′ UTR shortening. 


## 1. 引出反式作用
通过APA形成的广泛的3‘UTR缩短，在体内促进肿瘤生长 2。
> 2.Masamha, C. P. et al. CFIm25 links alternative polyadenylation to glioblastoma tumour suppression. Nature 510, 412–416 (2014).

一个流行的假说是，它顺式的介导原癌基因逃避miRNA的降解。
A prevailing hypothesis is that it induces proto-oncogene expression in cis through escaping microRNA-mediated repression. 

这里，我们报告一种惊人的3UTR缩短在某些转录本中富集，这些转录本被预测是抑癌基因的竞争性内源RNA( ceRNA )。
Here we report a surprising enrichment of 3′UTR shortening among transcripts that are predicted to act as competing-endogenous RNAs (ceRNAs) for tumor-suppressor genes.


## 2. 基于模型的反式作用

我们基于模型，对3’UTR缩短的反式作用分析，揭示了其在改变ceRNA表达量上的重要作用。
Our model-based analysis of the trans effect of 3′ UTR shortening (MAT3UTR) reveals a significant role in altering ceRNA expression. 

MAT3UTR预测了很多3‘UTR缩短的反式靶点，包括 PTEN(它是重要的抑癌基因，参与9个3UTR缩短的基因的ceRNA交谈，包括EPS15, NFIA)
MAT3UTR predicts many trans-targets of 3′ UTR shortening, including `PTEN`, a crucial tumor-suppressor gene 3 involved in ceRNA crosstalk 4 with nine 3′UTR-shortening genes, including EPS15 and NFIA. 
> 3.Zhang, S. & Yu, D. PI(3)king apart PTEN’s role in cancer. Clin. Cancer Res. 16, 4325–4330 (2010).
> 4.Poliseno, L. et al. A coding-independent function of gene and pseudogene mRNAs regulates tumour biology. Nature 465, 1033–1038 (2010).

敲低NUDT21(一个3‘UTR缩短调控子)，反式的以miRNA依赖的方式抑制抑癌基因，比如 PHF6和LARP1。
Knockdown of NUDT21, a master 3′ UTR-shortening regulator 2, represses tumor-suppressor genes such as PHF6 and LARP1 in trans in a miRNA-dependent manner. 

## 3. 结论
我们的结果分析表明，3’UTR缩短抑制抑癌基因，主要是通过反式打扰ceRNA交谈，而不是顺式诱导原癌基因。
Together, the results of our analysis suggest a major role of 3′ UTR shortening in repressing tumor-suppressor genes in trans by disrupting ceRNA crosstalk, rather than inducing proto-oncogenes in cis.






# Main
一段一个序号。

## 1.3UTR可能不是帮原癌基因逃逸miRNA降解促肿瘤的 
通过APA而3‘UTR缩短，通常出现在增强的细胞分裂和转化过程中。
> 5.Mayr, C. & Bartel, D. P. Widespread shortening of 3′UTRs by alternative cleavage and polyadenylation activates oncogenes in cancer cells. Cell 138, 673–684 (2009).
> 6.Fu, Y. et al. Differential genome-wide profiling of tandem 3′ UTRs among human breast cancer and normal cells by high-throughput sequencing. Genome Res. 21, 741–747 (2011).
> 7.Lin, Y. et al. An in-depth map of polyadenylation sites in cancer. Nucleic Acids Res. 40, 8460–8471 (2012).
> 8.Morris, A. R. et al. Alternative cleavage and polyadenylation during colorectal cancer development. Clin. Cancer 18, 5256–5266 (2012).

Recently, we reported that NUDT21-mediated 3′US promotes glioblastoma growth, further underscoring its significance to tumorigenesis 2. 

---
- 流行的观点是，缩短的3’UTR导致原癌基因逃脱了miRNA介导的抑制。
- 确实，几个被鉴定为原癌基因的，比如 CCND1 通过使用缩短的3UTR增加蛋白水平，但是多在细胞系中 5。
- 然而，在最近的 PolyA 测序 7 中，及我们的TCGA RNA测序APA分析中 1 (分别是5和358个 肿瘤/正常 样品对)，发现之前体外实验中 5 有3‘UTR缩短的原癌基因，在肿瘤中大多没有3UTR长度的改变。
> 1.Xia, Z. et al. Dynamic analyses of alternative polyadenylation from RNA-Seq reveal landscape of 3′ UTR usage across 7 tumor types. Nat. Commun. 5, 5274, (2014).

- 比如，我们从358肿瘤/正常样品对中，识别出 1346 反复出现的 3’UTR缩短(occurrence rate >20%)，然而，CCND1并不在列表中(8/358=2.2%)。




---
Q: 如何定义原癌基因?? //todo

Furthermore，和随机挑选的基因类似，3‘UTR缩短的基因，以及之前5个APA研究，都很少和前500(P<0.01)个高可信原癌基因重叠。

high-confidence oncogenes as defined on the basis of distinct somatic mutational patterns of >8,200 tumor/normal pairs 9.
> 9.Davoli, T. et al. Cumulative haploinsufficiency and triplosensitivity drive aneuploidy patterns and shape the cancer genome. Cell 155, 948–962 (2013).

These results challenge the previous hypothesis and suggest a different function of 3′US for tumorigenesis.







## 2. 假说 失去3UTR吸收的miRNA会直接抑制抑癌基因

3UTR除了顺式调控同源转录本，还可能参与反式ceRNA调控 10.
> 10.Salmena, L., Poliseno, L., Tay, Y., Kats, L. & Pandolfi, P. P. A ceRNA hypothesis: the Rosetta Stone of a hidden RNA language? Cell 146, 353–358 (2011).

ceRNA通常形成global regulatory networks (ceRNETs)，控制重要的生物学过程 <sup>11</sup>。
> 11.Thomson, D. W. & Dinger, M. E. Endogenous microRNA sponges: evidence and controversy. Nat. Rev. Genet. 17, 272–283 (2016).

For example, the tumor suppressor PTEN’s ceRNAs, CNOT6L and VAPA, have been shown to regulate PTEN and phenocopy its tumor-suppressive properties 12. 
> 12.Tay, Y. et al. Coding-independent regulation of the tumor suppressor PTEN by competing endogenous mRNAs. Cell 147, 344–357 (2011).

ceRNA调控轴主要基于3UTR的miRNA结合位点，我们的假说是，当基因3UTR缩短不能隔绝miRNA时，被释放的miRNA会直接 反式作用 抑制他们的ceRNA伴侣，比如 抑癌基因，从而导致肿瘤生成。


## 3. 检验假说
使用 97 个TCGA乳腺癌及配对正常样本，based on `miRNA-binding-site overlap and co-expression`13,14 ，使用 现有策略 搭建 2个 ceRNET。
> 13.Sumazin, P. et al. An extensive microRNA-mediated network of RNA–RNA interactions regulates established oncogenic pathways in glioblastoma. Cell 147, 370–381 (2011).
> 14.Ala, U. et al. Integrated transcriptional and competitive endogenous RNA networks are cross-regulated in permissive molecular environments. Proc. Natl Acad. Sci. USA 110, 7154–7159 (2013).

As expected, the loss of co-expression results in a much smaller (tenfold reduced) ceRNET for tumors than for normal tissues (Fig. 2b).

## 4. 研究3UTR缩短在ceRNET扰乱中的作用

- TCGA 乳腺癌中 ER阳性样本: 68/97，建立ceRNET网络。
- 使用 DaPars算法，发现 >20%肿瘤中出现的3US基因有427个。
- PTEN和EPS15有4个iRNA结合位点重合，在normal样本中是ceRNA对。
- 然而，EPS15在23/68(33.8%)肿瘤中有3US，失去和PTEN竞争miRNA的能力，在肿瘤中就和PTEN不共表达了。(fig.2c)

![PTEN-EPS15 co-expression](/data/2021/images/05/0521-NG-3UTR-ceRNA-PTEN_EPS15.png)
Fig. 2: 3′UTR shortening contributes to ceRNET disruption.
c, Gene expression of EPS15 (3′US gene) and PTEN (ceRNA partner) on 68 estrogen-receptor-positive (ERP) breast tumors and matched normal samples. The horizontal lines represent the mean expression values of PTEN, which is decreased in tumors (FDR = 2.1 × 10<sup>−10</sup>).


---
最显著3US的基因，前100个ceRNA都在肿瘤中失去互作(就是在normal中共表达，在tumor中依旧共表达)，而没有3US的100个ceRNA有12个保持了互作。
Globally, the top 100 ceRNAs with the most significant 3′US genes all lost their interactions in tumors, while 12 out of 100 ceRNAs lacking 3′US retained (P = 0.0002) their interactions. 


进一步在肿瘤-正常样本对中，最3US和最不3US的30个样本建立的ceRNET中，3US越狠损失的ceRNET越多。
Furthermore, in separate ceRNETs from 30 tumor/normal pairs with the least and most amount of 3′US (upper panel in Fig. 2d), more 3′US is clearly associated with more ceRNET loss (38.6 versus 16.4 in fold decrease, P < 1 × 10−16, lower panel in Fig. 2d).

![ceRNA-ERP](/data/2021/images/05/0521-NG-3UTR-ceRNA-ERP.png)
Fig.2d, The upper heatmap exhibits significant APA events (rows) across 68 ERP tumor/normal pairs (columns), ranked by the number of 3′US genes. The Venn diagrams show the number of ceRNA pairs in the normal and tumor ceRNET. The numbers in parentheses are normalized to the number of edges shared between tumor and normal tissues. The P value was calculated from a `one-tailed Pearson’s chi-squared test`.

这个P值怎么算的？至少R的函数帮助中没有设定 单尾 的地方。
```
> rs=chisq.test(matrix( c(204448, 12495, 20207, 215295, 5573, 7211), nrow = 2, byrow = T ) )
> rs

	Pearson's Chi-squared test

data:  matrix(c(204448, 12495, 20207, 215295, 5573, 7211), nrow = 2,     byrow = T)
X-squared = 8918.7, df = 2, p-value < 2.2e-16

> rs$p.value
[1] 0
```
数量级一致，但是数字是<2.2，文中是<1

---
所以，在肿瘤中，3US与ceRNA网络的破坏密切相关。
From these findings, we conclude that 3′US is strongly associated with ceRNA network disruption in tumors.


## 5. 3US反式抑制 抑癌基因表达
为研究3US介导的ceRNA扰乱的功能，选出381个3US的基因，从normal的ceRNET中选出2131个他们的ceRNA伴侣基因(3US ceRNA)，包括591个3US ceRNA hub及1540个3US ceRNA non-hub基因。
To understand the function of 3′US-mediated ceRNET disruption, we selected 381 3′US genes and 2,131 of their ceRNA partner genes (3′US ceRNAs), including 591 3′US ceRNA hub and 1,540 3′US ceRNA non-hub genes, in the normal ceRNET (Supplementary Table 1, Methods).

我们假设反式作用，3US的基因释放miRNA抑制其ceRNA伴侣。

---
符合我们的假说，2131个3US基因的ceRNA基因在肿瘤中是和其3US基因反相关的(r = −0.21; P = 5 × 10−24)
Consistent with our hypothesis, expression changes of 2,131 3′US ceRNA genes in tumors are anti-correlated (r = −0.21; P = 5 × 10−24) with the degree of 3′US of the associated 3′US genes (Supplementary Fig. 1a).

结果，肿瘤中976个normal ceRNET中的下调基因中，816(83.6%)个是3US基因的ceRNA。
As a result, among 976 genes in normal ceRNET downregulated in tumors, 816 (83.6%) are ceRNAs of 3′US genes. 

惊奇的是，3US ceRNA hub基因富集在抑癌基因(P ~ 1 × 10−20)，而不是原癌基因上(Fig.3a)，表明3US是反式抑制 抑癌基因的。
Surprisingly, 3′US ceRNA hub genes are enriched in tumor-suppressor genes (P ~ 1 × 10−20) but not in oncogenes (Fig. 3a), suggesting that the 3′US represses tumor suppressors in trans. 

比如，肿瘤中EPS15的3US就导致了其ceRNA伴侣PTEN的下调。
For example, 3′US of EPS15 would contribute to downregulating its ceRNA partner PTEN in tumors (Fig. 2c).


![fig3ab](/data/2021/images/05/0521-NG-3UTR-ceRNA-fig3ab.png)
Fig. 3: 3′UTR shortening represses tumor-suppressor genes in TCGA breast cancer.
**a**, Functional enrichment of 3′US ceRNA hub genes (red), 3′US ceRNA non-hub genes (blue), 3′US genes (purple) and random RefSeq genes (gray). We randomly sampled each gene category to the same number (381) 100 times; averaged P values with standard deviation are plotted. 
**b**, Relative expression (tumor/normal) of tumor-suppressor genes that are 3′US ceRNAs (n = 160, left box) is lower than for those that are not in ceRNET (n = 226, right box) (P = 8 × 10−3). 


---
全局上，160个3US ceRNA中的抑癌基因更倾向于下调，相对于226个非ceRNET中的对照抑癌基因(P = 8 × 10−3, Fig. 3b)，表明 3US和 抑癌基因的表达是显著相关的。
Globally, 160 expressed tumor-suppressor genes from 3′US ceRNAs are more likely downregulated than 226 control tumor-suppressor genes not in ceRNET (P = 8 × 10−3, Fig. 3b), indicating a significant association between 3′US and tumor-suppressor gene repression.






## 6. 为什么是这些基因3US，而不是其ceRNA伴侣抑癌基因?

对于序列的分析部分解答这个问题。

---
对TCGA 450K 甲基化芯片数据的分析表明，3US街道的ceRNA抑制，不依赖于启动子甲基化。


## 7.工具包 MAT3UTR: 反式作用是3US调控ceRNA基因表达的主要作用

该包需要4个数据: 3US基因的表达，3US的水平，miRNA结合位点，miRNA表达水平。
To better quantify the trans effects of 3′US, we developed a mathematical model (MAT3UTR) based on its 3′US gene(s) expression, 3′US level, miRNA-binding site(s) and miRNA expression(s) (Methods). 

- 在1548个差异表达的3US ceRNA中，MAT3UTR可以揭示47.6%的基因表达变异。
- 而对照模型，只考虑miRNA而不考虑3US，只能解释27.2%的变异，
- 这和之前的报道一致：miRNA自己对基因表达的调控是很弱的 17。
> 17.Hausser, J. & Zavolan, M. Identification and consequences of miRNA–target interactions — beyond repression of gene expression. Nat. Rev. Genet. 15, 599–612 (2014).


---
The results suggest that the trans effects of 3′US plays a major role in regulating ceRNA gene expression.



## 8. 3US的反式作用是PTEN下调的主要因素

MAT3UTR predicts many trans-target genes of 3′US, including PTEN, in ceRNA crosstalk11,12,13 (top 1% MAT3UTR score, Supplementary Table 2).
> 11.Thomson, D. W. & Dinger, M. E. Endogenous microRNA sponges: evidence and controversy. Nat. Rev. Genet. 17, 272–283 (2016)..
> 12.Tay, Y. et al. Coding-independent regulation of the tumor suppressor PTEN by competing endogenous mRNAs. Cell 147, 344–357 (2011).
> 13.Sumazin, P. et al. An extensive microRNA-mediated network of RNA–RNA interactions regulates established oncogenic pathways in glioblastoma. Cell 147, 370–381 (2011).

- normal ceRNET中，PTEN被预测为9个3US基因的ceRNA。
- 当对97个 乳腺癌-正常样品对，按照这9个基因的3US打分排序后，3US打分高的基因PTEN表达更低(P=0.03)
- Furthermore，MAT3UTR 可以解释肿瘤中PTEN表达变异的86.9%，表明在PTEN下调中，3US反式作用是主要因素。


![0521-NG-3UTR-ceRNA-fig3d.png](/data/2021/images/05/0521-NG-3UTR-ceRNA-fig3d.png)
Fig.3d, A heatmap in the top panel showing APA events for the nine 3′US genes (rows). The boxplots in the bottom panel show PTEN expression levels in 10 tumors with the most (left) or least (right) 3′ UTR shortening. 



## 9. 实验: EPS15下调则其ceRNA伴侣PTEN也下调

![0521-NG-3UTR-ceRNA-fig3efg.png](/data/2021/images/05/0521-NG-3UTR-ceRNA-fig3efg.png)
fig.3e: miRNA下调 MCF7细胞的EPS15后，wb条带显示PTEN也下调了，GAPDH做对照。
fig.3f: 荧光素酶报告基因，验证是不是依赖miRNA的靶点(PTEN的3UTR)：细胞裂解液中si-EPS15-2加入后，luc-PTEN 3UTR也下调了。
fig.3g: 是不是依赖miRNA? KD  si-Dicer酶，结果luc-PTEN 3UTR不下调了。
fig.3h,i: 过表达 EPS15 3UTR 能上调PTEN阳性的细胞。

Thus, EPS15 3′US may impact PTEN expression.




## 10. 回顾NUDT21 KD数据，3US导致抑癌基因下调是通过增加和AGO2复合体相关性实现的

To gain insights into the global cause-and-effect relationship between 3′US and the repression of tumor-suppressor genes, we revisited our previous data from `NUDT21-knockdown HeLa cells`, since NUDT21 is one of the master regulators of 3′US 2. 
> 2.Masamha, C. P. et al. CFIm25 links alternative polyadenylation to glioblastoma tumour suppression. Nature 510, 412–416 (2014).


- RISC (RNA-induced silencing complex) （RNA诱导沉默复合物）
- 仅依靠miRNA结合位点重合情况（因为2重复的共表达估算效力低），从NUDT21 KD细胞中找到1168个 3US ceRNA。
- 9914个与 3US没有显著miRNA重合的基因做对照，抑癌基因还是在3US ceRNA中强烈富集。
	* 3US ceRNA中的57个抑癌基因中，33个(57.9%)在NUDT21 KD样本中受抑制。
	* 而更小的比例44.5%(339*0.445 个)的对照 抑癌基因 受抑制。
- 说明 NUDT21 介导的3US 反式地抑制 抑癌基因。
- 虽然缺乏共表达导致ceRNA网络假阳性高，但是结果和TCGA 乳腺癌分析一致。

---
假说: 抑癌基因ceRNA的抑制，和RISC复合体中AGO2的被占用的增加有关。
On the basis of these results, we posit that repression of tumor-suppressor ceRNAs would correlate with increased occupancy of AGO2 in the RISC complex.

![0521-NG-3UTR-ceRNA-fig4cd.png](/data/2021/images/05/0521-NG-3UTR-ceRNA-fig4cd.png)
Fig. 4: NUDT21-mediated 3′ UTR shortening causes tumor-suppressor repression in trans.
**c**, Knockdown of NUDT21 in HeLa cells using CRISPR/Cas9 and reduced NUDT21 was detected by western blot analysis in three independent experiments. 
**d**, RIP was performed with anti-AGO2 antibody; normal mouse IgG served as a control. The RIP complexes were detected by western blot with a distinct AGO2 antibody from rat (inset). The indicated ceRNAs associated with AGO2 enrichment in NUDT21-knockdown cytoplasmic lysates versus the control are shown with average fold change ± standard deviation from three independent assays (P = 0.0002, P = 5.2 × 10−6, P = 0.0004, P = 0.0005, P = 0.0006, P = 0.01 and P = 5.47 × 10−7, two-sided t-test, * * P < 0.001, * P < 0.01).

> Q: 从NUDT21 KD 和 ctrl 细胞中提细胞质，使用anti-AGO2抗体做RNA IP(免疫沉淀)，怎么检测沉淀物中的抑癌基因的? 使用的 `qRT PCR`，方法学最后一页写了。


验证假说：
- 从ctrl和NUDT21 KD细胞分离细胞质组分，做使用anti-AGO2抗体做RNA免疫共沉淀。
- 平均，观察到在Ago2 RIP复合物中 ~200倍的ceRNA富集，相对于ctrl IgG.
- 降低NUDT21并不能影响 AGO2/DICER1 表达，和 GAPDH mRNA 结合到 AGO2.(fig. 4c,d)
- `miRNA seq`，ctrl vs NUDT21 KD，排除掉对miRNA生物合成的general effect，miRNA也是同样的上调或下调。(fig.S4d)
- 重要的是，在NUDT21删除后，能检测到多个抑癌基因ceRNA和AGO2相关性在1.5倍和7倍范围(Fig. 4d). 


--
3US导致抑癌基因减少，是通过增加和抑制性的AGO2复合体的相关性。
These results demonstrate that 3′US can lead to reduction of tumor-suppressor genes through their increased association with repressive AGO2 complexes.




## 11.另2个抑癌基因在NUDT21 KD后下调
- 抑癌基因 PHF6，LARP1及其ceRNA伴侣 YOD1，LAMC1。
- 在NUDT21 KD后，PHF6，LARP1下调，而 YOD1，LAMC1上调(fig.5a WB)。

--
为了验证是不是3UTR介导的这个效应
- 转了一个 包含PHF6 or LARP1的3UTR的荧光素酶报告基因，到 ctrl or NUDT21KD 细胞，检测荧光素酶活性。
- 发现 NUDT21 KD后，2个荧光素酶活性都下降了(fig.5b)。
- PHF6 or LARP1此前被认为是 抑癌基因 9,18,19，且在HeLa中下调能增加细胞的生长速度，验证他们是抑癌基因(fig.S5)
> 9.Davoli, T. et al. Cumulative haploinsufficiency and triplosensitivity drive aneuploidy patterns and shape the cancer genome. Cell 155, 948–962 (2013).
> 18.Mets, E. et al. MicroRNA-128-3p is a novel oncomiR targeting PHF6 in T-cell acute lymphoblastic leukemia. Haematologica 99, 1326–1333 (2014).
> 19.Selcuklu, S. D. et al. MicroRNA-9 inhibition of cell proliferation and identification of novel miR-9 targets by transcriptome profiling in breast cancer cells. J. Biol. Chem. 287, 29516–29528 (2012).


## 12. 实验: NUDT21介导的3US导致抑癌基因下调是miRNA依赖的
- 基于MAT3UTR分析和实验结果，选择 PHF6，研究抑癌基因下调的分子机制。
- 选择2个靶向PHF6的miRNA，方程(3)打分最高和第6高的。
- NUDT21 KD后，2个miRNA都没下调(fig.S4d)。
- 但是加miRNA拮抗剂后，抗 miR-549后 PHF6部分恢复；抗 miR-3187-3p 后 PHF6完全恢复(fig.5d)；
- 荧光素酶报告基因实验，Luc - PHF6 3UTR，加任意一个的拮抗剂，荧光部分恢复。


---
检验: 下调PHF6表达是否依赖YOD1水平？
- 转染YOD1 cDNA到YOD1 KD细胞中，=====> cDNA不包含3 UTR吗? //todo
- 发现重新表达YOD1并不能修复PHF6的表达(fig.5f)，
- 也不能恢复YOD1 3UTR介导的荧光素酶(fig.5g)，
- 表明，对PHF6的反式作用，是因为YOD1的3 UTR。


---
确定：PHF6和YOD1的crosstak是miRNA依赖的？
- 删除DICER1，就废除了 PHF6和YOD1的crosstalk(fig.5h)


---
Collectively, the data strongly suggest that NUDT21-mediated 3′US causes tumor-suppressor gene repression in trans in a miRNA-dependent manner.

## 13. 总结
- 有文献建议做 3US的ceRNA crosstalk 20-22
> 20.Tian, B. & Manley, J. L. Alternative cleavage and polyadenylation: the long and short of it. Trends Biochem. Sci. 38, 312–320 (2013).
> 21.Mueller, A. A., Cheung, T. H. & Rando, T. A. All’s well that ends well: alternative polyadenylation and its implications for stem cell biology. Curr. Opin. Cell Biol. 25, 222–232 (2013).
> 22.Li, L. et al. 3′UTR shortening identifies high-risk cancers with targeted dysregulation of the ceRNA network. Sci. Rep. 4, 5406 (2014).

我们队97对 TCGA乳腺癌 RNA-seq数据的分析，及之后的实验验证表明，3US 通过反式，抑制 抑癌基因，是广泛存在的。

---
虽然这个反式效应进一步强调了肿瘤进展中APA的作用，它也提供了基因调控的另一层，强调了进一步研究其他能干扰ceRNA crosstakl的潜在机制的需要，比如 RNA editing，以及与RBP的竞争。
While the trans effect further emphasizes the importance of APA in tumor progression, it also provides an additional layer of gene regulation and underscores the need for further investigation into other potential mechanisms23,24 that could perturb ceRNA crosstalk, such as RNA editing and competition with RNA-binding proteins.
> 23.Tay, Y., Rinn, J. & Pandolfi, P. P. The multilayered complexity of ceRNA crosstalk and competition. Nature 505, 344–352 (2014).
> 24.Wang, Y. et al. The emerging function and mechanism of ceRNAs in cancer. Trends Genet. 32, 211–224 (2016).



# Methods


## Tumor-suppressor genes and oncogenes 【抑癌基因和原癌基因list】

分析8200肿瘤和正常样品对，根据原癌基因激活位点，和抑癌基因失活突变，定义tumor-suppressor genes and oncogenes。
TUSON是一个分析肿瘤中突变模式，进而预测具体基因是抑癌基因或原癌基因的工具。
按照P值排序，前500个基因，去掉重复的30个，得到470个抑癌基因或原癌基因，用于本文的富集分析，超几何检验。

很少乳腺特有的抑癌基因和原癌基因(36个核3个q<0.05)，top500中90%是 泛癌预测。

- The tumor-suppressor genes and oncogenes used in this study were defined by the `TUSON algorithm` from genome sequencing of >8,200 tumor/normal pairs 9, namely residue-specific activating mutations for oncogenes and discrete inactivating mutations for tumor-suppressor genes. 
- TUSON is a computational method that analyzes patterns of mutation in tumors and predicts the likelihood that any individual gene functions as a tumor-suppressor gene or oncogene. 
- We ranked genes by their TUSON prediction P values from the most to the least significant and used the top 500 genes (P < 0.01) as the reference tumor-suppressor genes or oncogenes. 
- After removing 30 genes in common, 470 tumor-suppressor genes and oncogenes were used for the enrichment analysis. 

> 9.Davoli, T. et al. Cumulative haploinsufficiency and triplosensitivity drive aneuploidy patterns and shape the cancer genome. Cell 155, 948–962 (2013).

Note that there were `very few breast tumor-specific tumor-suppressor genes and oncogenes` (36 and 3 with breast q-value ≤ 0.5, respectively) and 90% of them were found in the top 500 pan-cancer predictions.



## Previously identified 3′US genes in cancers 之前发现的肿瘤中的3US的基因
- Xia et al. identified 1,187 3′US genes across 7 TCGA cancer types1. 
- Mayr and Bartel selected 23 3′US genes from 27 cancer cell lines5. 
- Fu et al. identified 428 3′US genes in human breast cancer cell lines6. 
- Lin et al. reported 120 3′US genes in major cancers and tumor cell lines7. 
- Morris et al. found 286 3′US genes in human colorectal tumor samples8. 

The 3′US genes of Xia et al. were randomly sampled to 100 genes for a fair comparison.


> 1.Xia, Z. et al. Dynamic analyses of alternative polyadenylation from RNA-Seq reveal landscape of 3′ UTR usage across 7 tumor types. Nat. Commun. 5, 5274, (2014).
> 5.Mayr, C. & Bartel, D. P. Widespread shortening of 3′UTRs by alternative cleavage and polyadenylation activates oncogenes in cancer cells. Cell 138, 673–684 (2009).
> 6.Fu, Y. et al. Differential genome-wide profiling of tandem 3′ UTRs among human breast cancer and normal cells by high-throughput sequencing. Genome Res. 21, 741–747 (2011).
> 7.Lin, Y. et al. An in-depth map of polyadenylation sites in cancer. Nucleic Acids Res. 40, 8460–8471 (2012).





## Selection of miRNA-binding sites 【汇编miRNA结合位点】

Predicted miRNA-binding sites were obtained from TargetScanHuman version 6.2 25. 

Only those with a preferentially conserved targeting score (Pct) more than 0 were used 1.

实验验证过的miRNA结合位点：
`Experimentally validated miRNA-binding sites` were obtained from 
- TarBase version 5.0 26, 
- miRecords version 4 27 
- and miRTarBase version 4.5 28. 

过滤掉间接方式获取的位点，如芯片实验或蛋白组学实验。
The binding sites found `in indirect studies` such as microarray experiments and high-throughput proteomics measurements were `filtered out` 29. 

另一个来源是从公开的AGO-CLIP数据构建出的miRNA靶点atlas中的显著位点(q<0.05):
`Another source is the microRNA target atlas` composed of public AGO-CLIP data 30 with significant binding sites (q-value <0.05). 

预测的和验证的位点信息，合并后用于本研究。
The predicted and validated binding site information was then combined to use in this study.

> 25.Lewis, B. P., Burge, C. B. & Bartel, D. P. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. Cell 120, 15–20 (2005).
> 26.Papadopoulos, G. L., Reczko, M., Simossis, V. A., Sethupathy, P. & Hatzigeorgiou, A. G. The database of experimentally supported targets: a functional update of TarBase. Nucleic Acids Res. 37, D155–D158 (2009).
> 27.Xiao, F. et al. miRecords: an integrated resource for microRNA–target interactions. Nucleic Acids Res. 37, D105–D110 (2009).
> 28.Hsu, S.-D. et al. miRTarBase update 2014: an information resource for experimentally validated miRNA–target interactions. Nucleic Acids Res. 42, D78–D85 (2014).
> 29.Dvinge, H. et al. The shaping and functional consequences of the microRNA landscape in breast cancer. Nature 497, 378–382 (2013).
> 30.Hamilton, M. P. et al. Identification of a pan-cancer oncogenic microRNA superfamily anchored by a central core seed motif. Nat. Commun. 4, 2730 (2013).


## TCGA breast tumor RNA-Seq and miRNA-Seq data 【下载TCGA 数据 RNAseq, miRNA-seq】

Quantified gene expression files (RNASeqV1) for primary breast tumors (TCGA sample code 01) and their matching solid normal samples (TCGA sample code 11) were downloaded from the TCGA Data Portal 31. 
> 31.Goldman, M. et al. The UCSC Cancer Genomics Browser: update 2013. Nucleic Acids Res. 41, D949–D954 (2013).
> https://genome-cancer.ucsc.edu/ -> http://xena.ucsc.edu/welcome-to-ucsc-xena/

//todo 怎么使用 xena 下载数据集?? //todo

---
97个样品对。10868个表达的RefSeq基因(FPKM>=1 in 80%样本中)用于下游分析。
We used 97 breast tumor samples that have matched normal tissues. A total of 10,868 expressed RefSeq genes (fragments per kilobase of transcript per million mapped reads (FPKM) ≥ 1 in >80% of all samples) were selected for downstream analyses. 

---
禁用CDS定量mRNA的表达。
To better quantify gene expression in the presence of 3′US, we used only coding regions (CDS) to quantify mRNA expression. 

Exon和CDS注释下载自 Sage Bionetworks’ Synapse 数据库. 
- https://www.synapse.org/ 
- https://cancercomplexity.synapse.org/Explore/Datasets
- https://cancercomplexity.synapse.org/
    - https://firecloud.terra.bio/
    - https://www.cri-iatlas.org/resources/
    - https://www.cbioportal.org/
Exon and CDS annotation for TCGA data and miRNA expressions (syn1445790) were downloaded from Sage Bionetworks’ Synapse database.

> 7个乳腺癌测序样品：https://cancercomplexity.synapse.org/Explore/Datasets
> Single Cell RNA sequencing of Adult Human Breast Epithelial Cells [C1_Individual_3]
> adult 1 https://cancercomplexity.synapse.org/Explore/Datasets/DetailsPage?datasetId=syn21795809
> adult 2 https://cancercomplexity.synapse.org/Explore/Datasets/DetailsPage?datasetId=syn21795292
> adult 3 https://cancercomplexity.synapse.org/Explore/Datasets/DetailsPage?datasetId=syn21792875
> Single Cell RNA sequencing of Adult Human Breast Epithelial Cells https://cancercomplexity.synapse.org/Explore/Datasets/DetailsPage?datasetId=syn21793531
> Single Cell RNA sequencing of Adult Human Breast Epithelial Cells [Individual 4..7] https://cancercomplexity.synapse.org/Explore/Datasets/DetailsPage?datasetId=syn21795279


## CeRNA identification in TCGA breast tumors

根据miRNA结合位点和表达量识别ceRNA。

> 13.Sumazin, P. et al. An extensive microRNA-mediated network of RNA–RNA interactions regulates established oncogenic pathways in glioblastoma. Cell 147, 370–381 (2011).
> 14.Ala, U. et al. Integrated transcriptional and competitive endogenous RNA networks are cross-regulated in permissive molecular environments. Proc. Natl Acad. Sci. USA 110, 7154–7159 (2013).

---
Q: 怎么计算miRNA结合位点的重合率的p值？ //todo

只选择中等表达量的miRNA，用于捕捉动态互作。
去掉少于6个miRNA结合位点的基因，选择基因对之间miRNA结合位点重合率显著(p.adj<0.05)。


- CeRNAs were identified by miRNA-binding-site overlap and expression correlation 13,14. 
- Only microRNAs with intermediate expression (between 0.01 and 100 in averaged fragments per million mapped fragments (FPM)) were used to capture dynamic interactions 14. 
- After removing genes with fewer than six such miRNA-binding sites, gene pairs with significant miRNA-binding-site overlap (<0.05 in Benjamini–Hochberg-corrected P value) were selected. 


---
Q: 相关系数>0.6是怎么来的? p值p<1e-10又是怎么来的?? //todo

还要满足表达量的相关系数>0.6, p<1e-10，才被认定为 ceRNA。
看率3UTR的变异，只用CDS定量mRNA表达量。
和>500 ceRNA 连接的基因被认定为hub 基因。

- Among them, `pairs correlated (>0.6 in Pearson’s correlation coefficient) (P < 1 × 10−10) in gene expression` were defined as ceRNAs. 
- To account for mRNAs with variable 3′ UTRs, we used only CDS to quantify mRNA expression. 
- Genes that are connected with >500 ceRNAs were defined as hub genes.


## Model-based analysis of trans effect of 3′US (MAT3UTR)
有很多公式，先跳过。





## Code availability
The open source MAT3UTR program (version 0.9.2) is freely available at https://github.com/thejustpark/MAT3UTR with necessary example data for this analysis.

## Data availability
Raw and processed miRNA-Seq data for the NUDT21-depletion experiment have been deposited to GEO under the accession number [GSE78198](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78198).








# 语言学习

underscore [ʌndə'skɔː]  vt. 强调；划线于……下
sequester [sɪˈkwestə(r)] vt. 使隔绝；使隐退；没收，扣押
permissive [pəˈmɪsɪv] dj. 许可的；自由的；宽容的；（两性关系）放纵的
disruption [dɪsˈrʌpʃn]  n. 扰乱，打乱，中断
empirically [ɪmˈpɪrɪkli] adv. 以经验为主地
posit [ˈpɒzɪt] vt. 安置；假定
abolish [əˈbɒlɪʃ]  vt. 废除，废止；取消，革除

underscore vt. 强调；划线于……下
V-T If something such as an action or an event `underscores` another, it draws attention to the other thing and emphasizes its importance. 突出显示; 强调 [美国英语]

tandem [ˈtændəm] n. 双人自行车；串联工作组 adj. 纵列的
compose [kəmˈpəʊz] vt. 构成；写作；使平静；排…的版



> 2021/5/28 End Reading.