
> **SCAPTURE: a deep learning-embedded pipeline that captures polyadenylation information from 3' tag-based RNA-seq of single cells**
> Genome Biol. 2021 Aug 10;22(1):221. doi: 10.1186/s13059-021-02437-5.
> https://pubmed.ncbi.nlm.nih.gov/34376223/
> https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8353616/
> https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-021-02437-5.pdf
> 本地: 2021-GB_SCUPTURE_s13059-021-02437-5.pdf
> https://github.com/YangLab/SCAPTURE


# take home message

核心代码使用了 tensorflow.keras：
https://github.com/YangLab/SCAPTURE/blob/main/DeepPASS/Predict.py


## 数据集
1. 10x 官网的6个PBMC数据集



2. (2019)  Systematic comparative analysis of single cell RNA-sequencing methods
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132044

> 17. Ding J, Adiconis X, Simmons SK, Kowalczyk MS et al. Systematic comparison of single-cell and single-nucleus RNA-sequencing methods. Nat Biotechnol 2020 Jun;38(6):737-746. PMID: 32341560


## 作图学习

Fig.S9D 免疫细胞的marker dotplot图。


# 图片

## Fig. 1 Developing SCAPTURE to identify cleavage and polyadenylation sites (PASs) from 3′ tag-based scRNAseq

分4个步骤。
- 使用UMI-tools来定量的。
>18. Smith T, Heger A, Sudbery I. UMI-tools: modeling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy. Genome Res. 2017;27(3):491–9. https://doi.org/10.1101/gr.209601.116.

2个特点：
- 使用了 深度神经网络 DeepPASS
- 定量化PAS位点，代替基因，用于识别细胞身份(cell identity)


---
Fig.1D ROC曲线看，比其他AI工具(DeepPASTA, APARENT)效果好。




## Fig. 2 Constructing the embedded DeepPASS model for position-insensitive prediction of PASs.

使用 shifting model 对位置不敏感的神经网络。




## Fig. 3 Applying SCAPTURE pipeline to study PASs in human PBMC datasets.



## Fig. 4 Using quantified PASs to represent distinct transcript expression for refining cell identities.


## Fig. 5 Identifying altered PAS usages upon SARS-CoV-2 infection.



# 正文











# 讨论

> 26.实验验证PAS位置。
Masamha CP, Xia Z, Yang J, Albrecht TR, Li M, Shyu AB, et al. CFIm25 links alternative polyadenylation to glioblastoma tumour suppression. Nature. 2014;510(7505):412–6. https://doi.org/10.1038/nature13261.

> 19.深度神经网络分析APA: DeepPASTA
Arefeen A, Xiao X, Jiang T. DeepPASTA: deep neural network based polyadenylation site analysis. Bioinformatics. 2019; 35(22):4577–85. https://doi.org/10.1093/bioinformatics/btz283.

> 20.深度神经网络分析APA: APARENT(本文作者认为AUC曲线不好: Fig.2D)
Bogard N, Linder J, Rosenberg AB, Seelig G. A deep neural network for predicting and engineering alternative polyadenylation. Cell. 2019;178(1):91–106 e123. https://doi.org/10.1016/j.cell.2019.04.046.





# 参考文献

> 24.综述文章
Gruber AJ, Zavolan M. Alternative cleavage and polyadenylation in health and disease. Nat Rev Genet. 2019;20(10):599–614. https://doi.org/10.1038/s41576-019-0145-z.

> 28.方法学 Sierra
Patrick R, Humphreys DT, Janbandhu V, Oshlack A, Ho JWK, Harvey RP, et al. Sierra: discovery of differential transcript usage from polyA-captured single-cell RNA-seq data. Genome Biol. 2020;21(1):167. https://doi.org/10.1186/s13059-020-02071-7.

> 29.方法学 scAPA（效果不好）
Shulman ED, Elkon R. Cell-type-specific analysis of alternative polyadenylation using single-cell transcriptomics data. Nucleic Acids Res. 2019;47(19):10027–39. https://doi.org/10.1093/nar/gkz781.

> 30.免疫细胞转录组，广泛存在的内含子APA(iPA)
Singh I, Lee SH, Sperling AS, Samur MK, Tai YT, Fulciniti M, et al. Widespread intronic polyadenylation diversifies immune cell transcriptomes. Nat Commun. 2018;9(1):1716. https://doi.org/10.1038/s41467-018-04112-z.

> 31.Seurat的文章
Stuart T, Butler A, Hoffman P, Hafemeister C, Papalexi E, Mauck WM 3rd, et al. Comprehensive integration of single-cell data. Cell. 2019;177(7):1888–902 e1821. https://doi.org/10.1016/j.cell.2019.05.031.

> 32.新冠数据集
Arunachalam PS, Wimmers F, Mok CKP, Perera R, Scott M, Hagan T, et al. Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans. Science. 2020;369(6508):1210–20. https://doi.org/10.1126/science.abc6261.


> 33-34: bulk 中，感染后激活近端PAS位点。
33.Jia X, Yuan S, Wang Y, Fu Y, Ge Y, Ge Y, et al. The role of alternative polyadenylation in the antiviral innate immune response. Nat Commun. 2017;8(1):14605. https://doi.org/10.1038/ncomms14605.
34.Pai AA, Baharian G, Page Sabourin A, Brinkworth JF, Nedelec Y, Foley JW, et al. Widespread shortening of 3' untranslated regions and increased exon inclusion are evolutionarily conserved features of innate immune responses to infection. PLoS Genet. 2016;12(9):e1006338. https://doi.org/10.1371/journal.pgen.1006338.






# 英语单词

bona fide  [ˌbəʊnə ˈfaɪdi] adj. 真实的；真诚的
corroborate [kəˈrɒbəreɪt] vt. 证实；使坚固




