1009-AML-cell-line.md

```
14株人类肿瘤细胞（HL60,Molm14,MV4-11,RS4:11,HepG2,Huh7,Skhep1,BT-549,MDM-BA231,MCF7,skBr3,H358,H1299以及Hela）

作者进一步研究了化合物11h对白血病细胞系（MV4-11、Molm14、RS4;11）和p53细胞系（U937、HL60）的抑制活性
```


# Blood: 怎么证明分化能力？
> Mitochondrial carrier homolog 2 is necessary for AML survival
> https://pdf.sciencedirectassets.com/778769/1-s2.0-S0006497120X6839X/1-s2.0-S000649712061919X/main.pdf


- `In AML, knockdown of MTCH2 decreased growth, reduced engraftment potential of stem cells, and induced differentiation.`
- ` Inhibiting MTCH2 in AML cells increased nuclear pyruvate and pyruvate dehydrogenase (PDH), which induced histone acetylation and subsequently promoted the differentiation of AML cells. `










# 细胞系 背景信息 / Cell line info

`OCI-AML2, HL60, K562, U937, and NB4` were cultured in Iscove modified Dulbecco medium and RPMI medium with 10% fetal calf serum, 100 U/mL streptomycin, and 100 mg/mL penicillin, respectively.

`8227 cells` were provided by John E. Dick (Princess Margaret Cancer Centre) and were cultured in X-VIVO-10 with 20% bovine serum albumin, insulin, and transferrin;


- 构建细胞系

OCI-AML2-Cas9 cell generation OCI-AML2 cells were transduced with Cas9-expressing lentiviral vector, Lenti‐Cas9‐2A‐Blast (Addgene plasmid #73310; Watertown, MA),16 and selected with blasticidin (10 mg/mL) for 6 days.
After selection, the Cas9-expressing clonal population was subject to 2 rounds of single-cell cloning. From the second round, a designated clone-7.2 was obtained, characterized by immunoblotting and used for the screening assay.



## 表观修饰：甲基化 / ATAC /

- https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97346 芯片: 呼吸链抑制剂处理24h后
Gene expression of AML cell lines (HL60, KG1a, MOLM14 and U937) untreated or treated with metformin

- 化疗耐药 https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78102 (Fig 3E, KM plot dataset 2; Cell line HL60/MOLM14/KG1a/U937/KG1/MV4-11)


- https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200454 NA-seq of MV4;11 cells treated with Methyl Bestatin or Brefeldin A
- 甲基 苯丁抑制素 methyl bestatin (MeBs, aminopeptidase inhibitor) or 
- 布雷菲德菌素 brefeldin A (BFA, ER-to-Golgi transport inhibitor), compared to DMSO control.


- https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE151249
GSM4569608	MV4-11-5_RNA seq
GSM4569609	MV4-11-11_RNA seq


- GSE137647: ATAC-seq analysis of K562, MKPL-1, MV4-11, NB4 and U937 leukemia cell lines




## AML 细胞系 简介

> https://www.researchgate.net/post/Which-cell-line-is-most-suitable-for-AML

Loucy and OCI-AML1 cell lines

We have been developing a drug against AML and used human U937 monocytic cell line and HL-60 as AML modell. 
As AML mouse modell I recommend you to use C-1498 cell , RAW 264.7 or M1 murine Cell Line.


---

- The K562 line is composed of undifferentiated blast cells that are rich in glycophorin and may be induced to produce fetal and embryonic hemoglobin in the presence of hemin.
K562系由富含糖蛋白的未分化母细胞组成，在血红蛋白存在的情况下可诱导产生胎儿和胚胎血红蛋白。

- The KG-1 cell line is composed predominantly of myeloblasts and promyelocytes. A unique characteristic of the KG-1 cells is their almost complete dependence on colony-stimulating factor for proliferation in soft-gel culture.
KG-1细胞系主要由成髓细胞和早幼髓细胞组成。KG-1细胞的一个独特特征是它们在软凝胶培养中几乎完全依赖集落刺激因子增殖。

- The `HL-60` is a promyelocytic leukemia cell line.  In the presence of DMSO, the cells mature into granulocytes.
HL-60是早幼粒细胞白血病细胞系。在DMSO存在下，细胞成熟为粒细胞。

- Both the KG-1 and HL-60 cells differentiate into nondividing mononuclear phagocytes when exposed to phorbol esters. 
KG-1和HL-60细胞在暴露于磷酯时均分化为不分裂的单核吞噬细胞。



---

- https://www.sciencedirect.com/science/article/pii/S0006291X19312069

AML cell lines (Kas-1, `HL60`, NB4 and `K562`) and normal cell line of 293T were obtained from American Type Culture Collection (ATCC, USA). 

The effects of TYP on AML were also confirmed in vivo using BALB/c mice xenografts bearing HL60 cells. 

---


- https://www.sciencedirect.com/topics/medicine-and-dentistry/acute-myeloid-leukemia-cell-line

The various vitamin D compounds have similar effects on inducing differentiation and inhibiting proliferation of several `acute myeloid leukemia cell lines` such as `HL-60, U937, THP-1, HEL, and NB4`.

In contrast, `more immature myeloid leukemia cell lines` such as `HL-60 blasts, KG1, KGla, and K562` do not respond to the hormone.

Vitamin D analogs inhibit cell growth mainly by inducing cell cycle arrest. Many studies have shown that treated leukemic cell lines accumulate in the G0/G1 and G2/M phase of the cell cycle, with a concomitant decrease in the proportion of cells in S-phase [31–33].

HL-60 cells treated with 1,25(OH)2D3 acquire the morphology and functional characteristics of `macrophages`. Expression of the cell-surface markers CD14 and CD11b are up-regulated. 





## 正常细胞系？

> https://www.sciencedirect.com/science/article/pii/S2211124723012093?via%3Dihub
> The normal colorectal cell lines (`NCM460`) and colorectal cancer cell lines (`SW480`, `RKO`) were cultured in RPMI 1640 supplemented with 1% penicillin-streptomycin and 10% FBS.

那么，AML 有没有对应的 normal cell line?



> AML cell lines with a list of expression markers: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10049680/


## Kasumi-3

The Kasumi-3 cell line was derived from a 57-year-old male patient with M0. It has a few notable characteristics, including a doubling time of approximately 55–60 h in cell culture, an appearance consistent with immature lymphoblasts, and the expression of various markers, as shown in Table 1 [11]. B cell-specific, T-ALL, and NK markers were also identified in this cell line [12]. 

The Kasumi-3 engraftment model in NSG mice was also developed. Mice were implanted with 1 × 106 or 10 × 106 per mouse. The amount of human CD45+ cells in peripheral blood, blood marrow, and spleen was monitored by FACS to estimate the disease progression. 


KG-1 and KG-1a are challenging to differentiate in vitro, making them valuable for studying potential drugs that promote differentiation, such as differentiating CD34+ progenitor cells into dendritic-like cells [35,36].



## Kasumi-1

Cell line: kasumi-1(急性髓系白血病细胞): 人急性原粒细胞白血病细胞

This cell line expresses myeloperoxidase and can differentiate into neutrophils and macrophages after treatment with IL-5 or TPA, respectively. It also expresses `CD34`, suggesting it originated from an early myeloid stem cell. 




## Jurkat (人急性T细胞白血病细胞): T-ALL

- https://www.cobioer.com/product/1583.html

Established from the peripheral blood of a 14-year-old boy with acute lymphoblastic leukemia (ALL) at first relapse in 1976 (in the original paper by Schneider et al. the cell line was labelled as JM)

Cell type:	T cell leukemia

Morphology:	round cells growing singly or in clumps in suspension

- https://baijiahao.baidu.com/s?id=1757438441811659853&wfr=spider&for=pc

人白血病 T 淋巴细胞 Jurkat, Clone E6-1 由 Schneider 建立，来源于一个 14 岁的男孩的外周血。

Jurkat细胞是一种常用的人类T细胞白血病细胞系，这种人类T细胞白血病细胞系是生物学研究的宝贵资源。这种细胞系具有快速增殖能力和易于培养的特点，因此在生物学研究中占有重要地位。

Jurkat细胞是一种悬浮细胞。用于研究急性T细胞白血病，T细胞信号传导以及易感病毒进入的各种趋化因子受体，特别是HIV的表达的永生化人T淋巴细胞系。它在生物学研究上具有十分广泛的应用，比如应用Jurkat细胞研究核糖核酸酶P的M1-RNA，探讨抗MHCⅡ类分子转录激活因子(CⅡTA)的M1-RNA抑制细胞表面MHCⅡ类分子的表达。







## ATCC: Leukemia Cell Line Panel

- https://www.atcc.org/products/tcp-1010

A panel of 7 leukemia cell lines representing Acute Myeloid Leukemia (AML), Acute Monocytic Leukemia (AMoL), Chronic Myeloid Leukemia (CML), and Acute Lymphoblastic Leukemia (ALL). Each culture contains genomic mutations in one or more of the following genes according to the Sanger COSMIC database: CDKN2A, KDM6A, TP53, NRAS, NOTCH1, PTEN, FBXW7, FLT3, KRAS, MLH1, and PIK3R1.

```
Components

KASUMI-1, ATCC CRL-2724
HL-60, ATCC CCL-240
THP-1, ATCC TIB-202
K-562, ATCC CCL-243
RS4;11, ATCC CRL-1873
MOLT-4, ATCC CRL-1582 : T-cell acute lymphoblastic leukemia (T-ALL) 
CCRF-CEM, ATCC CCL-119
```




## THP-1细胞系

THP-1细胞系是从一名患有急性单核细胞白血病的1岁小男孩的外周血中分离得到的，自1980年建系以来，THP-1细胞被广泛用于单核细胞和巨噬细胞相关的机制、信号通路以及营养和药物运输等研究中。
相对于U937、HL-60、ML-2等白血病细胞系，THP-1更有类似人原代单核细胞的形态和功能特征（包括细胞分化标记）。相对于人外周血单核细胞（PBMC），THP-1更易在实验室中培养和扩增，且具有更稳定的基因背景，不存在PBMC的个体差异性问题，利于实验结果的重现。因此，THP-1是各大实验室常用的急性单核细胞白血病细胞系，是研究免疫和炎症的理想工具。


















# 细胞系 比对信息 (map to hg38 /bam format)


## Control: 293T (IGV)

```
1. download

https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE151296
SRR11858511 2.84Gb  293T GSM4572153  wild type
SRR11858512 2.68Gb  293T GSM4572154  wild type

$ cd /home/wangjl/data/rsa/293T/
$ fasterq-dump --split-files SRR11858511

2. map
$ mkdir map

id="SRR11858511";

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/293T/${id}_1.fastq /data/wangjl/rsa/293T/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/293T/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/293T/map/ /data/wangjl/HeLa/cell_cycle/293T
```




## Control: hESC H1 (IGV)

```
1. down
1) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE188111
Genome-wide transcriptomic profiling of cardiomyocyte differentiation from Wild-type and PERK-KO H1 hESCs [RNA-seq]
PERK 是否敲除; 分化诱导 day1,2,3,5,10;

GSM5670253	H1_Day0_rep1 Day0 SRR16768631 2Gb
GSM5670254	H1_Day0_rep2 Day0 SRR16768632 2Gb
GSM5670255	H1_Day0_rep3 Day0 SRR16768633 2Gb

GSM5670268	H1_Day10_rep1 SRR16768646 3Gb
GSM5670269	H1_Day10_rep2 SRR16768647
GSM5670270	H1_Day10_rep3 SRR16768648


2) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE181049 酒精和干细胞发育 //先不用
Lim Y, Beane-Ebel JE, Tanaka Y, Ning B et al. Exploration of alcohol use disorder-associated brain miRNA-mRNA regulatory networks. Transl Psychiatry 2021 Oct 2;11(1):504. PMID: 34601489

GSM5482571	Ethanol-unexposed rep 1  
GSM5482572	Ethanol-unexposed rep 2  

$ cd /home/wangjl/data/rsa/hESC/
$ fasterq-dump --split-files SRR16768646


2. map
$ mkdir map
id="SRR16768646";

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/hESC/${id}_1.fastq /data/wangjl/rsa/hESC/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/hESC/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/hESC/map/ /data/wangjl/HeLa/cell_cycle/hESC
```



## control: hSperm (IGV) 效果不好

```
1. down
(1) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40196 读段太少
GSM987942	Sperm Donor1 mRNA-seq SRR543691 1Gb
GSM987943	Sperm Donor2 mRNA-seq SRR543695 0.3Gb

(2) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134580
identify the population of sperm RNAs responding to obesity as measured by BMI.

SRR9719710 healthy GSM3956402 1Gb
SRR9719718 overweight GSM3956410 0.5Gb
SRR9719729 obese GSM3956421 0.9Gb

(3) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129216 ncRNA-Seq 跳过
The effects of di-butyl phthalate exposure from medications on human sperm RNA among men
邻苯二甲酸二丁酯

GSM3702499	Sample_6_shortRNA
GSM3702500	Sample_9_shortRNA

GSM3702584	Sample_6_longRNA
GSM3702587	Sample_9_longRNA

(4) Highly conserved sperm function-related transcripts across three species: human, rat and mouse.
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE173659

GSM5273207	human sperm_Sample H7  SRR14370448 4.5Gb
GSM5273208	human sperm_Sample H1  SRR14370449
GSM5273209	human sperm_Sample H12 SRR14370450
GSM5273210	human sperm_Sample H13 SRR14370451
GSM5273211	human sperm_Sample H18 SRR14370452
GSM5273212	human sperm_Sample H19 SRR14370453

$ cd /home/wangjl/data/rsa/hSperm/
$ fasterq-dump --split-files SRR14370448
下载2个


2. map
$ mkdir map
id=SRR14370448

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/hSperm/${id}_1.fastq /data/wangjl/rsa/hSperm/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/hSperm/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/hSperm/map/ /data/wangjl/HeLa/cell_cycle/hSperm
```





## control: nerve (效果很差)

```
1. down
(1) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881
Transcriptional profiling of adult retinal ganglion cells during optic nerve regeneration

retinal [ˈretɪnl] adj. 视网膜的
ganglion [ˈɡæŋɡliən] n. [组织] 神经节；[医] 腱鞘囊肿

GSM4247141	Normal RGC sample 1 SRR10821187 1Gb
GSM4247142	Normal RGC sample 2 SRR10821188
GSM4247143	Normal RGC sample 3 SRR10821189

$ cd /home/wangjl/data/rsa/nerve/
$ fasterq-dump --split-files SRR10821187


2. map
$ mkdir map
id=SRR10821187

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/nerve/${id}_1.fastq /data/wangjl/rsa/nerve/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/nerve/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/nerve/map/ /data/wangjl/HeLa/cell_cycle/nerve
```



## control brain: BT142, TS603 

搜索: human brain cell line
找到 https://www.jianshu.com/p/86a1c2da26ee

```
1. down
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138858

We report the levels of the transcripts arising from the RNA sequencing analysis of `IDH1-mutant cell lines`
grade III oligoastrocytoma

GSM4120502	BT142_1 SRR10278840 2.3Gb
GSM4120503	BT142_2 SRR10278841
GSM4120504	BT142_3 SRR10278842

GSM4120505	TS603_1 SRR10278843
GSM4120506	TS603_2 SRR10278844
GSM4120507	TS603_3 SRR10278845

$ cd /home/wangjl/data/rsa/BT142/
$ fasterq-dump --split-files SRR10278840

$ cd /home/wangjl/data/rsa/TS603/
$ fasterq-dump --split-files SRR10278843


2. map
$ mkdir map
id=SRR10278840

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/BT142/${id}_1.fastq /data/wangjl/rsa/BT142/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/BT142/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/BT142/map/ /data/wangjl/HeLa/cell_cycle/BT142



---

2. map
$ mkdir map
id=SRR10278843

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/TS603/${id}_1.fastq /data/wangjl/rsa/TS603/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/TS603/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/TS603/map/ /data/wangjl/HeLa/cell_cycle/TS603
```





## Jurkat (T-ALL cell line) (IGV)

- T细胞激活前后 https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218256

- 氧气压力下甲基化的变化 https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE199235

```
1. downlowd	

(1) 单端 https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE220995
Comparative gene expression profiling analysis of RNA-seq data for Jurket T cells with and in the absence of FLU treatment.

GSM6836669	Jurkat T cells incubated with vehicle, 4 hrs 1 SRR22741428 1.8Gb
GSM6836670	Jurkat T cells incubated with vehicle, 4 hrs 2 SRR22741427
GSM6836671	Jurkat T cells incubated with vehicle, 4 hrs 3 SRR22741426

(2) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135370

GSM4006973	n1  SRR9905122 0.6Gb
GSM4006977	n2  SRR9905126 0.8Gb
GSM4006981	n3  SRR9905130 1.4Gb

$ cd /home/wangjl/data/rsa/Jurkat/
$ fasterq-dump --split-files SRR22741426

2. map
$ mkdir map
id=SRR22741426

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/Jurkat/${id}.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/Jurkat/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/Jurkat/map/ /data/wangjl/HeLa/cell_cycle/Jurkat
```





## MOLT-4 (T-ALL cell line)(IGV)

```
1. download
GSM5993284	Molt-4 PRMT7 WT repl. 1 SRR18574394 4.8Gb
GSM5993285	Molt-4 PRMT7 WT repl. 2 SRR18574395 3.6Gb
GSM5993286	Molt-4 PRMT7 WT repl. 3 SRR18574396 3.5Gb

$ cd /home/wangjl/data/rsa/MOLT-4/
$ fasterq-dump --split-files SRR18574394

2. map
$ mkdir map
id=SRR18574394

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/MOLT-4/${id}_1.fastq /data/wangjl/rsa/MOLT-4/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/MOLT-4/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/MOLT-4/map/ /data/wangjl/HeLa/cell_cycle/MOLT-4
```


## KG1 //no data

```

```


## TEX (IGV)

> SYK inhibition targets acute myeloid leukemia stem cells by blocking their oxidative metabolism
> https://www.nature.com/articles/s41419-020-03156-8
> SYK phosphorylation was markedly reduced by R406 in all primary AML samples tested (n = 7) and in `TEX, KG1, and MOLM14` cell lines (Fig. 1A, B, D).


---

> Small-molecule inhibitors targeting Polycomb repressive complex 1 RING domain
> https://www.nature.com/articles/s41589-021-00815-5
> 1. `TEX` cells are derived from hematopoietic stem and progenitor cells transformed with the TLS-ERG oncogene and express `high levels of CD34 and low levels of CD38`, which serve as LIC markers29,30.
> 2. all of which are consistent with differentiation of TEX cells
> 3. Leukemic stem cells (LSCs) or `leukemia-initiating cells (LICs)` represent a rare population of cells that are capable of self-renewal, proliferation and differentiation into malignant blasts13,14. 


```
1. down
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103834
GSM2782576	TEX_No_virus_1 SRR6033649 3.6Gb 252bp
GSM2782577	TEX_No_virus_2 SRR6033650

$ cd /home/wangjl/data/rsa/OCI-AML3/
$ fasterq-dump --split-files SRR6033649

2. map
$ mkdir map
id=SRR6033649

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/TEX/${id}_1.fastq /data/wangjl/rsa/TEX/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/TEX/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/TEX/map/ /data/wangjl/HeLa/cell_cycle/TEX
```



## OCI-AML3 (IGV)

https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE209518

we performed unbiased bulk RNA sequencing comparison between OCI-AML3 wt and CD123 and or CD33 KO clones

```
1. download

GSM6373730	1_AML3_wt SRR20396321 2.4Gb
GSM6373731	2_AML3_wt SRR20396320 7Gb
GSM6373732	3_AML3_wt SRR20396319 4Gb

$ cd /home/wangjl/data/rsa/OCI-AML3/
$ fasterq-dump --split-files SRR20396319


2. map
$ mkdir map
id=SRR18574394

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/OCI-AML3/${id}_1.fastq /data/wangjl/rsa/OCI-AML3/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/OCI-AML3/map/${id}_

3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/OCI-AML3/map/ /data/wangjl/HeLa/cell_cycle/OCI-AML3
```








## HEL (IGV): AML M6

- Description HEL - 
- Cell line: HEL 
- DSMZ no.: ACC 11
- Species validation: human (Homo sapiens)
- Cell type: erythroleukemia 
- Origin: established from the peripheral blood of a 30-year-old man with erythroleukemia (AML M6)


```
1. download
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214480

GSM6607143	HEL cell, shCtrl_biol rep 1  SRR21758080 平均 2.5Gb
GSM6607144	HEL cell, shCtrl_biol rep 2  SRR21758079 
GSM6607145	HEL cell, shCtrl_biol rep 3  SRR21758078

$ cd /home/wangjl/data/rsa/HEL/
$ fasterq-dump --split-files SRR21758078
2023-10-20T06:41:47 fasterq-dump.3.0.7 err: insp_get_file_size( 'https://sra-pub-run-odp.s3.amazonaws.com/sra/SRR21758078/SRR21758078', remotely = YES ) -> RC(rcNS,rcFile,rcCreating,rcTimeout,rcExhausted)


2. map
$ mkdir map
id="SRR21758078";

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/HEL/${id}_1.fastq /data/wangjl/rsa/HEL/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/HEL/map/${id}_


3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;


4.View on scIGV
$ ln -s /data/wangjl/rsa/HEL/map/ /data/wangjl/HeLa/cell_cycle/HEL
```





## 分化 Kasumi-1 & NB4 (IGV)

We observed that the compounds budesonide, mometasone and halcinonide drastically differentiate the CD34+ primitive cells to CD15+ terminally differentiated blasts (Fig. 2a–c).


- In the present study, we focused on the human Akirin2 regulome and interactome in `neutrophil-like model human Caucasian promyelocytic leukemia HL60 cells`. 
promyelocytic 早幼粒细胞的


```
1. 下载
1) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE210200

GSM6423427	Kasumi-1 control_1  SRR20729564
GSM6423428	Kasumi-1 control_2  SRR20729563
GSM6423429	Kasumi-1 control_3  SRR20729562

GSM6423433	NB4 treated with control_1  SRR20751819
GSM6423434	NB4 treated with control_2  SRR20751818

平均 7-9G/sample

2) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146475
RNA-sequencing and 3'RNA-sequencing (3'READS) of Kasumi-1 cells transduced with two independent FIP1L1-targeting shRNAs or a control shRNA.

GSM4386836	shControl_rep1_RNA-seq  SRR11247521 2Gb
GSM4386837	shControl_rep2_RNA-seq  SRR11247522
GSM4386838	shControl_rep3_RNA-seq  SRR11247523
GSM4386839	shControl_rep4_RNA-seq  SRR11247524

GSM4386840	shFIP1L1-1_rep1_RNA-seq SRR11247525
GSM4386841	shFIP1L1-1_rep2_RNA-seq SRR11247526
GSM4386842	shFIP1L1-1_rep3_RNA-seq SRR11247527
GSM4386843	shFIP1L1-1_rep4_RNA-seq SRR11247528

GSM4386844	shFIP1L1-2_rep1_RNA-seq SRR11247529
GSM4386845	shFIP1L1-2_rep2_RNA-seq SRR11247530
GSM4386846	shFIP1L1-2_rep3_RNA-seq SRR11247531
GSM4386847	shFIP1L1-2_rep4_RNA-seq SRR11247532

$ cd /home/wangjl/data/rsa/kasumi-1
$ fasterq-dump --split-files SRR11247525




2. 比对
/data/wangjl/rsa/kasumi-1/
$ mkdir map/

id=SRR11247521

1)kasumi-1
STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/kasumi-1/${id}_1.fastq /data/wangjl/rsa/kasumi-1/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/kasumi-1/map/${id}_


/data/wangjl/rsa/NB4/
$ mkdir map/

2) NB4
id="SRR20751818";
id="SRR20751819";

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/NB4/${id}_1.fastq /data/wangjl/rsa/NB4/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/NB4/map/${id}_


3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/kasumi-1/map/ /data/wangjl/HeLa/cell_cycle/kasumi-1
$ ln -s /data/wangjl/rsa/NB4/map/ /data/wangjl/HeLa/cell_cycle/NB4
```


基因表达定量

```
$ cd /home/wangjl/data/rsa/kasumi-1/
$ mkdir counts

$ featureCounts \
-a /data/wangjl/ref/hg38/gencode/GRCh38.p13.gtf \
-p \
-T 10 \
-o counts/featureCounts_SRR1124_matrix2.txt \
-t exon -g gene_name \
map/SRR1124*.sortedByCoord.out.bam #12个

仅比对到 54%-65%!!
每一行是 FAM138A，基因名。

然后使用R t-test 比较DEG: kd vs. wt. 
作者认为 kd 后促进白血病细胞分化为髓系，接近正常。
```



### DaPars of RNA-seq data after FIP1L1 KD

DaPars 的使用，参考 http://blog3.dawneve.com/#/post/2023/1005-APA-cell-cycle.md

```
1. 准备3个文件
(2)To generate the BedGraph files from BAM files

$ cd /home/wangjl/data/rsa/kasumi-1
$ mkdir bedGraph
$ bedtools genomecov -bg -ibam /home/wangjl/data/rsa/kasumi-1/map/SRR11247521_Aligned.sortedByCoord.out.bam \
-g /home/wangjl/data/ref/hg38/hg38_chr_size.txt -split > /home/wangjl/data/rsa/kasumi-1/bedGraph/SRR11247521.bedgraph


$ cat SRR_list.txt 
SRR11247521
SRR11247522
SRR11247523
SRR11247524
SRR11247525
SRR11247526
SRR11247527
SRR11247528
SRR11247529
SRR11247530
SRR11247531
SRR11247532


$ cat SRR_list.txt | head -n 4|tail -n4 | while read id; do echo $id `date`; 
$ cat SRR_list.txt | head -n 8|tail -n4 | while read id; do echo $id `date`; 
$ cat SRR_list.txt | tail -n4 | while read id; do echo $id `date`; 
bedtools genomecov -bg -ibam /home/wangjl/data/rsa/kasumi-1/map/${id}_Aligned.sortedByCoord.out.bam \
-g /home/wangjl/data/ref/hg38/hg38_chr_size.txt -split > /home/wangjl/data/rsa/kasumi-1/bedGraph/${id}.bedgraph;
done;




2. 开始运行 DaPars (Z server)
(2). Sample processing
The files generated in step 1 above will be used in step 2. 

$ cd /home/wangjl/data/rsa/kasumi-1/bedGraph/
$ cat configure_file_KD1_WT.config
########################
#(1)3UTR annotation: The following file is the result of step 1.
Annotated_3UTR=/home/wangjl/data/ref/hg38/hg38_refseq_extracted_3UTR.bed 

#(2)sample bedGraph: A comma-separated list of BedGraph files of samples from condition 1: KO
Group1_Tophat_aligned_Wig=SRR11247521.bedgraph,SRR11247522.bedgraph,SRR11247523.bedgraph,SRR11247524.bedgraph

#A comma-separated list of BedGraph files of samples from condition 2:WT
Group2_Tophat_aligned_Wig=SRR11247525.bedgraph,SRR11247526.bedgraph,SRR11247527.bedgraph,SRR11247528.bedgraph

Output_directory=DaPars_Output/
Output_result_file=FIP1L1_KD1_WT

#At least how many samples passing the coverage threshold in two conditions
Num_least_in_group1=1
Num_least_in_group2=1

Coverage_cutoff=20

#Cutoff for FDR of P-values from Fisher exact test.
FDR_cutoff=0.05
PDUI_cutoff=0.2
Fold_change_cutoff=0.59
########################


$ cat configure_file_KD2_WT.config
########################
#(1)3UTR annotation: The following file is the result of step 1.
Annotated_3UTR=/home/wangjl/data/ref/hg38/hg38_refseq_extracted_3UTR.bed 

#(2)sample bedGraph: A comma-separated list of BedGraph files of samples from condition 1: KO
Group1_Tophat_aligned_Wig=SRR11247521.bedgraph,SRR11247522.bedgraph,SRR11247523.bedgraph,SRR11247524.bedgraph

#A comma-separated list of BedGraph files of samples from condition 2:WT
Group2_Tophat_aligned_Wig=SRR11247529.bedgraph,SRR11247530.bedgraph,SRR11247531.bedgraph,SRR11247532.bedgraph

Output_directory=DaPars_Output/
Output_result_file=FIP1L1_KD2_WT

#At least how many samples passing the coverage threshold in two conditions
Num_least_in_group1=1
Num_least_in_group2=1

Coverage_cutoff=20

#Cutoff for FDR of P-values from Fisher exact test.
FDR_cutoff=0.05
PDUI_cutoff=0.2
Fold_change_cutoff=0.59
########################


$ python2 ~/software/dapars/src/DaPars_main.py configure_file_KD1_WT.config
# 11:23->12:20

$ python2 ~/software/dapars/src/DaPars_main.py configure_file_KD2_WT.config
# 11:23->12:20



3. 过滤检查
$ cd /home/wangjl/data/rsa/kasumi-1/bedGraph/
-rw-rw-r-- 1 wangjl wangjl 2.3M Nov 10 12:20 FIP1L1_KD1_WT_All_Prediction_Results.txt
-rw-rw-r-- 1 wangjl wangjl 2.3M Nov 10 12:21 FIP1L1_KD2_WT_All_Prediction_Results.txt

$ cat DaPars_Output/FIP1L1_KD1_WT_All_Prediction_Results.txt | awk '{print $34}' | sort | uniq -c
   9135 N
      1 Pass_Filter
    136 Y

$ cat DaPars_Output/FIP1L1_KD2_WT_All_Prediction_Results.txt | awk '{print $34}' | sort | uniq -c
   9403 N
      1 Pass_Filter
     47 Y

(2) 手工核对
Gene    Loci    Pass_Filter     adjusted.P_val  PDUI_Group_diff

$ cat DaPars_Output/FIP1L1_KD1_WT_All_Prediction_Results.txt | awk '$34=="Y"{print $1"\t"$4"\t"$34"\t"$33"\t"$31}' | sort -k 5nr | wc
    136     680   10587

$ cat DaPars_Output/FIP1L1_KD1_WT_All_Prediction_Results.txt | awk '$34=="Y"{print $1"\t"$4"\t"$34"\t"$33"\t"$31}' | sort -k 5nr | head
NR_003700.1|PI4KAP2|chr22|-     chr22:21472998-21473405 Y       4.60808885704e-05       0.465
NM_001206855.3|TFR2|chr7|-      chr7:100620420-100621126        Y       0.00014234316245        0.4475
NM_020441.3|CORO1B|chr11|-      chr11:67435510-67438501 Y       1.70243795829e-13       0.43
NM_001145544.2|ZSCAN18|chr19|-  chr19:58083856-58085376 Y       0.0114602192433 0.3925
NM_023926.5|ZSCAN18|chr19|-     chr19:58083856-58085379 Y       0.0114602192433 0.3925
XM_005259174.6|ZSCAN18|chr19|-  chr19:58083842-58085379 Y       0.0114602192433 0.3875
NM_015465.5|GEMIN5|chr5|-       chr5:154887411-154888377        Y       0.000542394041626       0.385
NM_001354944.2|CUL4A|chr13|+    chr13:113229642-113230219       Y       0.00133378685772        0.3825
XM_047440354.1|DHX35|chr20|+    chr20:39038499-39039721 Y       0.000643750019056       0.3625
NM_001003931.4|PARP3|chr3|+     chr3:51948311-51948867  Y       0.000740072138544       0.36

$ cat DaPars_Output/FIP1L1_KD1_WT_All_Prediction_Results.txt | awk '$34=="Y"{print $1"\t"$4"\t"$34"\t"$33"\t"$31}' | sort -k 5nr | tail
XM_005256564.5|MPRIP|chr17|+    chr17:17184823-17192643 Y       0.000183005615811       -0.4025
NM_000849.5|GSTM3|chr1|-        chr1:109733937-109737169        Y       1.5727772605e-16        -0.42
NM_001354845.2|CCNB1|chr5|+     chr5:69177524-69178245  Y       0.000515949691511       -0.4275
NM_148955.4|SNX1|chr15|+        chr15:64137568-64144231 Y       4.45519867668e-11       -0.445
XM_047424329.1|LOC124900275|chr9|+      chr9:129504892-129512670        Y       8.68503642711e-06       -0.4475
NM_025205.5|MED28|chr4|+        chr4:17623601-17629306  Y       3.29718872152e-12       -0.4575
XR_007095652.1|ANKRD28|chr3|-   chr3:15667236-15668029  Y       4.76997106203e-07       -0.4775
NM_002871.5|RABIF|chr1|-        chr1:202878282-202881223        Y       0.00251004945504        -0.4875
NM_172208.3|TAPBP|chr6|-        chr6:33303606-33303989  Y       2.81486719942e-06       -0.5575
NM_001282293.2|ABCB8|chr7|+     chr7:151045209-151047782        Y       3.72749822733e-05       -0.59

- PI4KAP2 
- TFR2   不明显
- CORO1B 重叠基因区域
- ZSCAN18 exp low;
- GEMIN5 不明显
- CUL4A 3utr long;
- DHX35 不明显
- PARP3 近端 **

- MPRIP long 3UTR 8k; 不明显
- GSTM3 distal pA; ***
- CCNB1 distal pA; **
- SNX1 distal pA; ??
- LOC124900275
- MED28 very long 3UTR, 21k; distal pA; **
- ANKRD28 
- RABIF 
- TAPBP 
- ABCB8 
```







### 3'READS of FIP1L1 shRNA in Kasumi-1 (路径有改变)
```
1. https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146475 SE,75bp
GSM4386848	shControl_rep1_3'READS SRR11247533
GSM4386849	shControl_rep2_3'READS SRR11247534
GSM4386850	shControl_rep3_3'READS SRR11247535

GSM4386851	shFIP1L1-1_rep1_3'READS SRR11247536
GSM4386852	shFIP1L1-1_rep2_3'READS SRR11247537
GSM4386853	shFIP1L1-1_rep3_3'READS SRR11247538
GSM4386854	shFIP1L1-1_rep4_3'READS SRR11247539

GSM4386855	shFIP1L1-2_rep1_3'READS SRR11247540
GSM4386856	shFIP1L1-2_rep2_3'READS SRR11247541
GSM4386857	shFIP1L1-2_rep3_3'READS SRR11247542

$ cd /home/wangjl/data/rsa/kasumi-1/3READS/
$ fasterq-dump --split-files SRR11247533


2.map
$ cd /home/wangjl/data/rsa/kasumi-1/3READS/
$ mkdir map
id=SRR11247533

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/kasumi-1/3READS/${id}_1.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFilterMatchNminOverLread 0.2 --outFilterScoreMinOverLread 0.2 \
--outFileNamePrefix /data/wangjl/rsa/kasumi-1/3READS/map/short_${id}_

仅 63% uniq mapping，too short 21%.
添加参数 20/75=0.2，只要比对上20个就行，默认是0.66。 umiq mapping 73%
0.13 是 9bp，uniq mapping 73%



3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4.View on scIGV
$ ln -s /data/wangjl/rsa/kasumi-1/3READS/map/ /data/wangjl/HeLa/cell_cycle/kasumi-1_3READS
```











## 8227 cell line: //No data;


## Molm14 //todo

MOLM-14人急性髓系白血病细胞系分离自人急性髓系白血病的外周血，包含免疫B细胞和免疫T细胞，主要用于体外急性髓系白血病细胞的培养研究。

```
1. 下载 

1) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149520
RNA-seq analysis of human KMT2A rearranged Molm14 AML cell line treated with DOT1L and/or EZH2 inhibitor

GSM4504320	DMSO-1 SRR11634441 2Gb
GSM4504324	DMSO-2 SRR11634445 
GSM4504328	DMSO-3 SRR11634449
GSM4504332	DMSO-4 SRR11634453


2. 

```



## U937 (IGV)

```
1. 下载
1a)https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE184324

The AML cell line U937 was treated with decitabine (DAC) ± ATRA in technical triplicates for 3 consecutive days (with daily change of media and DAC add-on). 
Cells were harvested and RNA was isolated after 72 hours, Fragment analysis by TapeStation confirmed RQN values ≥9.7.
rRNA was removed and strand-specific libraries of each replicate were sequenced on an Illumina HiSeq 2500 with approximately 40 million reads per sample.

GSM5584108	U937_untreated_1 SRR15929145 6Gb
GSM5584109	U937_untreated_2 SRR15929146
GSM5584110	U937_untreated_3 SRR15929147

$ fasterq-dump --split-files SRR15929145

1b)https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE223609
Comparative gene expression profiling by performing RNA-seq in 3 replicates each of control U937 cells and its knockdown derivatives (shKDM6A, shKDM6B and shKDM6A+6B)

GSM6965474	U937 cells, shControl, RNA-seq Replicate 1  SRR23203934  9Gb
GSM6965475	U937 cells, shControl, RNA-seq Replicate 2  SRR23203933
GSM6965476	U937 cells, shControl, RNA-seq Replicate 3  SRR23203932

/data/wangjl/rsa/U937/
$ scp wangjl@y.biomooc.com:/data/wangjl/sra/cell_cycle/SRR*fastq . 

2. map
$ mkdir map

id="SRR15929145";
id="SRR15929146";
id="SRR15929147";

id="SRR23203932";
id="SRR23203933";
id="SRR23203934";

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/U937/${id}_1.fastq /data/wangjl/rsa/U937/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/U937/map/${id}_


3. index
ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4. view
$ ln -s /data/wangjl/rsa/U937/map/ /data/wangjl/HeLa/cell_cycle/U937
```





## THP-1 cell line (macrophage) (IGV)

- https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE234829
- Role of Carboxylesterase 1 in THP-1 Human Macrophages Under Basal Conditions or Treated with LPS+IFN Gamma (M1) OR with IL4 (M2)

> 尝试研究 APA 变化： Macrophage 极化过程中

- M1 which is bactericidal and pro inflammatory, 
- and M2 which is anti inflammatory and active in tissue repair.

```
1. 下载

1) 单端(不能用，太差)
GSM4305745	THP-1 cells, control A  SRR11059463  1.8Gb # 失败：峰图太差，也很乱，不能看。原因未知。
GSM4305746	THP-1 cells, control B  SRR11059464  1.7Gb # 失败：峰图太差，也很乱，不能看。原因未知。

2) 换一个来源？ https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE234829
# resting (M0), 
# LPS + interferon gamma (M1), 
# and IL4 (M2), each analyzed in quadruplicate.
# 还有 CES1 KD 的没下载

GSM7475290	WT_THP-1 cells M0_1 SRR24910839 平均2Gb
GSM7475291	WT_THP-1 cells M0_2 SRR24910838
GSM7475292	WT_THP-1 cells M0_3 SRR24910837
GSM7475293	WT_THP-1 cells M0_4 SRR24910836

GSM7475294	WT_THP-1 cells M1_1 SRR24910835
GSM7475295	WT_THP-1 cells M1_2 SRR24910834
GSM7475296	WT_THP-1 cells M1_3 SRR24910833
GSM7475297	WT_THP-1 cells M1_4 SRR24910832

GSM7475298	WT_THP-1 cells M2_1 SRR24910831
GSM7475299	WT_THP-1 cells M2_2 SRR24910830
GSM7475300	WT_THP-1 cells M2_3 SRR24910829
GSM7475301	WT_THP-1 cells M2_4 SRR24910828


$ fasterq-dump --split-files SRR24910828
$ fasterq-dump --split-files SRR24910832

id="SRR24910836"; ls -lth ${id}*

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/THP-1/${id}_1.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/THP-1/map/${id}_

$ ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

$ ln -s /data/wangjl/rsa/THP-1/map/ /data/wangjl/HeLa/cell_cycle/THP-1
```



## MV4-11 cell line (IGV)

```
1. 下载
1) 深度太低了，且单端
GSM4306624	MV-4-11 control A  SRR11067848  677Mb
GSM4306625	MV-4-11 control B  SRR11067849  608Mb

/home/wangjl/data/rsa/MV4-11/
$ fasterq-dump --split-files SRR26016545

2) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE242849
GSM7771964	MV4-11 NC-1  SRR26016545 8.1Gb
GSM7771965	MV4-11 NC-2  SRR26016544 7.8Gb

//未下载
3) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149251 
GSM4494920	CTR-SP
GSM4494921	CTR-MP

4) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178287
GSM5387027	MV4-NC-1
GSM5387028	MV4-NC-2






2. map
$ mkdir map

id="SRR26016544"; ls -lth ${id}*

STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/MV4-11/${id}_1.fastq /data/wangjl/rsa/MV4-11/${id}_2.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/MV4-11/map/${id}_

3. index
$ ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;

4. view
$ ln -s /data/wangjl/rsa/MV4-11/map/ /data/wangjl/HeLa/cell_cycle/MV4-11
```




## HL60 cell line (IGV)

- https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE158885

HL-60 cells are promyeoloblasts isolated from the peripheral blood by leukopheresis from a 36-year-old,White, female with acute promyelocytic leukemia. This cell line can be used in immune disorder and immunology research.

```
1. 下载数据
1) 效果不好，在3‘端末尾有一个高峰，莫名其妙!

平均5G/sample;
Run	AvgSpotLen	cell_line	genotype	GEO_Accession (exp)
SRR12759747	300	HL60	wild type	GSM4813900
SRR12759748	300	HL60	wild type	GSM4813901
SRR12759749	300	HL60	wild type	GSM4813902
SRR12759750	300	HL60	AKR2 knock down	GSM4813903
SRR12759751	300	HL60	AKR2 knock down	GSM4813904
SRR12759752	300	HL60	AKR2 knock down	GSM4813905

AKIRIN2, not AKR2;

Y 传送到 Z:
Z@ /data/wangjl/rsa/HL60/
$ scp wangjl@y.biomooc.com:/data/wangjl/sra/cell_cycle/SRR12759748* .
平行传2个文件，总和还是 10M/s。


2) 换一个来源
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE193094

GSM5773115	HL-60_RNA_rep1  SRR17446841  4Gb
GSM5773118	HL-60_RNA_rep2  SRR17446838  2Gb
GSM5773121	HL-60_RNA_rep3  SRR17446835  1.5Gb

$ fasterq-dump --split-files SRR17446841


2. 比对: 单端
id="SRR17446841";
id="SRR17446838";
id="SRR17446835";


STAR --runThreadN 50 \
--outSAMtype BAM SortedByCoordinate \
--genomeDir /data/wangjl/ref/hg38/gencode/index/STAR/ \
--readFilesIn /data/wangjl/rsa/HL60/${id}_1.fastq \
--genomeLoad LoadAndKeep \
--limitBAMsortRAM 20000000000 \
--outFileNamePrefix /data/wangjl/rsa/HL60/map/${id}_

3. index
/data/wangjl/rsa/HL60
$ ls  map/*bam | while read id; do echo $id; 
samtools index -@ 10 $id;
done;


4. View
$ ln -s /data/wangjl/rsa/HL60/map/ /data/wangjl/HeLa/cell_cycle/HL60
```








# APA seq method

These have been extensively reviewed elsewhere [19,97] and include, for example, `PAS-seq, 3′-seq and 3′READS`, which also allow analysis of rare or FACS-purified cell populations.



# HSC -> MPP -> Mature blood cells

> Fig.3 APA Is Differentially Regulated in Stem Cell Hierarchies.
> (B) `HSCs exhibit rather long 3′-UTRs`, and shortening is observed in the MPP compartment. 
In mature blood cells, globally longer 3′-UTRs have been detected. 
No direct quantitative comparison between HSCs/MPPs and mature blood cells is possible at this point, as the studies have been performed independently. 




# 稳态？ cellular homeostasis？怎么研究的？

https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157583


# AML APA 分化

https://pubmed.ncbi.nlm.nih.gov/34482400/






# CD34+ dataset //todo

## 分离方法

- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6889502/

Briefly, mononuclear cells were isolated using Ficoll (GE Healthcare) and were enriched 60–90% for CD34+ hematopoietic and progenitor cells (HSPCs) (STEM CELL Technologies).


- Colony formation assay
- Cell cycle and apoptosis
	* To discriminate between G0/G1, cells were fixed and permeabilized using the BD Cytofix/Cytoperm kit (BD Biosciences).







## (1) GSE92274 Microwell-Seq

- https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92274
- https://www.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA357074&o=acc_s%3Aa

> Lai S, Huang W, Xu Y, Jiang M et al. Comparative transcriptomic analysis of hematopoietic system between human and mouse by Microwell-seq. Cell Discov 2018;4:34. PMID: 29977598

```
We sampled 
7 batches (CD34+_b1-7) of Human CD34+ mobilized peripheral blood stem cells (PBSCs); 
2 batches (CD34+_thawed_l6-7) of thawed Human CD34+ mobilized peripheral blood stem cells (PBSCs) and 
2 batches (CD34-_b1-2) of thawed Human CD34- mobilized peripheral blood stem cells (PBSCs), 
around 5000 single cells per sample.

Human CD34+ cells were selected by human CD34 selection kit (StemCell Technologies Cat #18056), following the protocol provided by the manufacturer. After spin at 300 x g for 5min, the supernatant was removed and CD34+ cells were suspended in 1x DPBS + 2mM EDTA and diluted to ~ 100,000/mL for Microwell-seq. 

After thawing in IMDM medium, human CD34+ cells were suspended in 100μL of PBS+5% FBS for exposure of antibodies.


下载2个: 来自外周血的CD34+细胞。
SRR5099310 4Gb  /Human CD34 positive cells  GSM2424886  CD34+_b6
SRR5099311 6Gb /Human CD34 positive cells  GSM2424887	CD34+_b7
SRR5099312 6Gb  /Human CD34 negative cells  GSM2424888  CD34-_1
SRR5217574 1Gb  /Human CD34 negative cells  GSM2424889	CD34-_2

$ fasterq-dump --split-files SRR5099310
$ fasterq-dump --split-files SRR5099312

$ fasterq-dump --split-files SRR5217574
$ fasterq-dump --split-files SRR5099311

@Z server
$ scp wangjl@y.biomooc.com:/data/wangjl/sra/cell_cycle/cd34/*fastq .



这个是
```



## (2) Microwell-seq 2

- https://pubmed.ncbi.nlm.nih.gov/32977829/

> J Hematol Oncol. 2020 Sep 25;13(1):128. doi: 10.1186/s13045-020-00941-y.
> A single-cell survey of cellular hierarchy in acute myeloid leukemia

```
Using Microwell-seq, a high-throughput single-cell mRNA sequencing platform, we analyzed the cellular hierarchy of bone marrow samples from 40 patients and 3 healthy donors. We also used single-cell single-molecule real-time (SMRT) sequencing to investigate the clonal heterogeneity of AML cells.
```



## (3) GSE144568 (引用很多GSE)

- https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03031-7#availability-of-data-and-materials
```
Targeted DNA sequencing and scRNA-seq data used in this study were deposited into the Genome Sequence Archive for Human at the BIG data center, Beijing Institute of Genomics, Chinese Academy of Sciences and China National Center for Bioinformation under accession number HRA001009 [84], HRA000996 [85], HRA001021 [86]. 
The release of these data is permitted by The Ministry of Science and Technology of the People’s Republic of China (permission number 2023BAT0907). 

Published scRNA-seq datasets of healthy donors were available under GEO accession `GSE144568` [10], `GSE120221` [11], `GSE132509` [12], `GSE116256` [21]. 

Bulk RNA-seq data of AML patients were obtained from TCGA [87] and TARGET [88] website. 

The clinical and genetic information were retrieved from their uploaded supplementary Table S1 [57] and the NCI TARGET website [89] for TCGA and TARGET, respectively. 

Published microarray expression dataset for flow cytometry-sorted CD34+CD38− cells from AML patients was obtained under the GEO accession `GSE76008` [30]. 

The authors declared that all other data supporting the findings of the study are within the paper and its additional files.
```


## GSE144568 (6 healthy)
- https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144568

We used 10x genomics platform to profile single-cell transcriptome of CD34+Lineage- hematopoietic stem/progenitor cells (HSPCs) derived from 16 human myelofibrosis tissue samples and 5 healthy donors


## GSE120221 (20 healthy, 2018)


## GSE132509: ALL (2020: childhood leukemia)
> Caron M, St-Onge P, Sontag T, Wang YC et al. `Single-cell analysis of childhood leukemia reveals a link between developmental states and ribosomal protein expression as a source of intra-individual heterogeneity`. Sci Rep 2020 May 15;10(1):8079. PMID: 32415257

Assessing intra-individual transcriptional heterogeneity in cALL patients using single cell RNA-seq

10X Genomics 3’ single cell RNA-seq: 

- Pre-B t(12;21) [ETV6-RUNX1] acute lymphoblastic leukemia, 
- Pre-B High hyper diploid [HHD] acute lymphoblastic leukemia, 
- Pre-T acute lymphoblastic leukemia, 
- Healthy pediatric bone marrow mononuclear cells. 

Annotations.tsv contains sample, cell cycle phase, tSNE/UMAP coordinates, and assigned cell type.

```
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132509

GSM3872434	ETV6-RUNX1_1  SRR9264343
GSM3872435	ETV6-RUNX1_2  SRR9264344
GSM3872436	ETV6-RUNX1_3  SRR9264345
GSM3872437	ETV6-RUNX1_4  SRR9264346
GSM3872438	HHD_1  SRR9264347
GSM3872439	HHD_2  SRR9264348
GSM3872440	PRE-T_1  SRR9264349
GSM3872441	PRE-T_2  SRR9264350

GSM3872442	PBMMC_1  SRR9264352
GSM3872443	PBMMC_2  SRR9264353
GSM3872444	PBMMC_3  SRR9264354

平均 17-26Gb

```




# AML 与 线粒体氧化磷酸化（oxidative phosphorylation，OXPHOS）

https://new.qq.com/rain/a/20230221A02DPF00

https://www.yxj.org.cn/detailPage?articleId=387235


- https://zhuanlan.zhihu.com/p/611206560?utm_id=0 线粒体蛋白亚定位方法



- AML细胞(尤其是LSCs)高度依赖线粒体氧化磷酸化(OXPHOS)存活，但OXPHOS过度活跃的机制尚不清楚，也缺乏抑制OXPHOS的非细胞毒性策略。
- G0, G1, G2-S 怎么区分？
- 在PDX（人源性肿瘤异种移植）小鼠模型上，作者发现敲低MFN2/OPA1导致癌细胞更难在小鼠身上生长
- 作者进一步探究了线粒体融合调控AML增殖和细胞周期的分子机制。通过对于线粒体功能的研究，作者发现敲低MFN2/OPA1导致ROS（活性氧物质）降低。使用活性氧清除剂可以达到MFN2/OPA1敲低一样的增加G0、减少G1细胞比例的效果。


急性髓系白血病（AML）是髓系原始细胞分化障碍及异常扩增导致的血液癌症，也是儿童中继ALL（急性淋系白血病）之后第二常见的白血病。尽管儿童AML疗效在近几十年内有了极大提高，但五年总体生存率仍然只有70%左右。

线粒体是为细胞提供能量的重要细胞器，与癌细胞增殖有密切关系。靶向线粒体蛋白质BCL2的维奈托克与化疗联用取得了良好的AML临床疗效，进一步表明了线粒体相关细胞过程与AML有密切关系。BCL2调控的细胞凋亡仅仅是线粒体的众多功能之一，线粒体的其他功能很可能与AML相关，有可能在线粒体中发现AML的新靶点。

近日，来自瑞士的 Tamburini 团队在 Leukemia 上发表了题为 Mitochondrial fusion is a therapeutic vulnerability of acute myeloid leukemia 的文章。文章发现了抑制线粒体融合可以导致`AML癌细胞在G0/G1细胞周期停滞`，是潜在的AML新治疗方法。



维奈托克是一种一流的靶向药物，旨在选择性地结合和抑制B细胞淋巴瘤-2（BCL-2）蛋白。

在一些血癌和其他肿瘤中，BCL-2会积聚并阻止癌细胞死亡或自毁，这一过程称为细胞凋亡。维奈托克能阻断BCL-2蛋白，帮助恢复细胞凋亡过程。

靶向线粒体蛋白质BCL2的维奈托克与化疗联用取得了良好的AML临床疗效，进一步表明了线粒体相关细胞过程与AML有密切关系。BCL2调控的细胞凋亡仅仅是线粒体的众多功能之一，线粒体的其他功能很可能与AML相关，有可能在线粒体中发现AML的新靶点。



# 亚细胞定位：GFP

目前研究亚细胞定位最常用的是融合报告基因定位法，将目标蛋白与荧光蛋白的N端或者C端融合，通过瞬时转化技术或稳定遗传转化技术，使得该融合蛋白在受体材料细胞内表达，目标蛋白会牵引荧光蛋白一起定位到目标细胞器，在扫描共聚焦显微镜的激光照射下会发出绿色荧光，从而可以精确地定位蛋白质的位置。



